youtube_dl.go

```diff
@@ -91,8 +91,9 @@ func (dl *AudioTrack) Play() {
 		if !isNil(dl.playlist) {
 			message = fmt.Sprintf(message+`<tr><td align="center">From playlist "%s"</td></tr>`, dl.Playlist().Title())
 		}
-		dj.client.Self.Channel.Send(message+`</table>`, false)
-
+		if dj.conf.General.AnnounceNewTrack == true {
```

Comment:
Small nitpick: could you change this line to the following?

`if dj.conf.General.AnnounceNewTrack {`